stable/app/templates/migration-job.yaml

```diff
@@ -10,7 +10,12 @@ spec:
     metadata:
       {{- with .Values.podAnnotations }}
       annotations:
-        {{- toYaml . | nindent 8 }}
+      # This is what defines this resource as a hook. Without this line, the
+      # job is considered part of the release.
+      "helm.sh/hook": post-install
+      "helm.sh/hook-weight": "-5"
+      "helm.sh/hook-delete-policy": hook-succeeded
```

Comment:
This means that only if the job completes successfully we are deleting the job. For failed job we have to manually delete the job always

Reply:
Can we make hook `pre-install` and delete for both failed and successful jobs